tests/async/test_page.py

```diff
@@ -30,7 +30,7 @@ async def test_close_should_reject_all_promises(context):
     assert "Protocol error" in exc_info.value.message
 
 
-async def test_closed_should_not_visible_in_context_pages(context):
+async def test_closed_should_not_visible_in_context_pages(context, browser):
```

Comment:
whats the reason for that?

Reply:
i was testing the version to see if channel works... reverting